2.6/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM php:7.2-cli-alpine3.7
+
+LABEL maintainer="frank.giesecke@final-gene.de"
+
+ENV PHPMD_VERSION 2.6.0
+
+WORKDIR /opt
+
+RUN apk add --no-cache --virtual=.build-deps bash
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+RUN curl -LSs "https://github.com/phpmd/phpmd/archive/${PHPMD_VERSION}.tar.gz" | tar -xz
+
+WORKDIR /opt/phpmd-${PHPMD_VERSION}
+
+RUN curl -LSs http://getcomposer.org/installer | php \
```

Comment:
Why?

Reply:
I've decide to use the source code as it's not possible to download a specific version of the provided PHAR.
Therefor it's necessary to run `composer install` which temporary requires the `composer` binary.